src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -145,7 +145,18 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
             bool isNullableAnalysisEnabled,
             BindingDiagnosticBag diagnostics) :
             base(containingType, syntax.GetReference(), location, isIterator: false,
-                 MakeModifiersAndFlags(property, propertyModifiers, isNullableAnalysisEnabled))
+                MakeModifiersAndFlags(
```

Comment:
The goal of this change is to make it so the getter symbol in `public virtual T Prop => ...` is not treated as introducing any of its own modifiers.

This makes the behavior of APIs like `LocalAccessibility` more consistent with properties declared using the accessor list syntax `public virtual T Prop { get => ... }`.